java/ql/src/Likely Bugs/Comparison/BitwiseSignCheck.ql

```diff
@@ -16,4 +16,4 @@ where
   e.isStrict() and
   e.getGreaterOperand() instanceof BitwiseExpr and
   e.getLesserOperand().(IntegerLiteral).getIntValue() = 0
-select e, "Sign check of a bitwise operation."
+select e, "Potential unsafe sign check of a bitwise operation."
```

Comment:
Shouldn't that be
```suggestion
select e, "Potentially unsafe sign check of a bitwise operation."
```

Reply:
Either works gramatically, with different bindings:

Potential (unsafe check)
(Potentially unsafe) check

Reply:
Yeah, I just felt that the latter was closer to the intended message. But I guess both work semantically.